From: Alessandro Bono <alessandro.bono369@gmail.com>
Date: Wed, 8 May 2024 16:06:17 +0200
Subject: info: Fix incompatible pointer type
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

This fixes the following:
```
libfreerdp/core/info.c: In function ‘rdp_read_info_null_string’:
libfreerdp/core/info.c:88:39: error: initialization of ‘const WCHAR *’ {aka ‘const short unsigned int *’} from incompatible pointer type ‘BYTE *’ {aka ‘unsigned char *’} [-Wincompatible-pointer-types]
   88 |                 const WCHAR* domain = Stream_Pointer(s);
```

(cherry picked from commit 4f411197dc9d2076f00748b1178a60b2423030bf)
---
 libfreerdp/core/info.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libfreerdp/core/info.c b/libfreerdp/core/info.c
index 9aaa6cf..c9b2fc6 100644
--- a/libfreerdp/core/info.c
+++ b/libfreerdp/core/info.c
@@ -85,7 +85,7 @@ static BOOL rdp_read_info_null_string(const char* what, UINT32 flags, wStream* s
 
 	if (cbLen > 0)
 	{
-		const WCHAR* domain = Stream_Pointer(s);
+		const WCHAR* domain = (WCHAR*)Stream_Pointer(s);
 
 		if (isNullTerminated && (max > 0))
 			max -= nullSize;
